api-operator/deploy/crds/wso2_v1alpha1_api_crd.yaml

```diff
@@ -87,9 +89,32 @@ spec:
           - definition
           type: object
         status:
+          properties:
+            replicas:
+              description: 'INSERT ADDITIONAL STATUS FIELD - define observed state
+                of cluster Important: Run "operator-sdk generate k8s" to regenerate
+                code after modifying this file Add custom validation using kubebuilder
+                tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html'
```

Comment:
Let's change this description

Reply:
updated

Reply:
Let's add this description in the comment section of Replicas of the file `api-operator/pkg/apis/wso2/v1alpha1/api_types.go`
Since this is a auto generated file, we may lost this description when generating this file again.

Let's edit the comment in here.
https://github.com/wso2/k8s-api-operator/blob/c801e64f7dda78473ab866f3df9d23a623ed5db5/api-operator/pkg/apis/wso2/v1alpha1/api_types.go#L43-L47